mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -1713,21 +1702,22 @@ impl TendermintCoin {
             // non-zero values are error.
             match tx_response.code {
                 TX_SUCCESS_CODE => Ok(Some(cosmrs::tendermint::abci::Code::Ok)),
-                err_code => Ok(Some(cosmrs::tendermint::abci::Code::Err(err_code))),
+                err_code => Ok(Some(cosmrs::tendermint::abci::Code::Err(
+                    NonZeroU32::new(err_code).unwrap(),
+                ))),
```

Comment:
Can you add comment here that this will never panic as `0` goes to the first `TX_SUCCESS_CODE` match?